getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -331,50 +333,51 @@ Save the following in sof-dev.conf.
   BUILD_NOCLEAN = 1
 
 
-For Fedora and derived distributions, make the following changes:
+For targets running Fedora and derived distributions, make the following changes:
 
 .. code-block:: perl
 
-  GRUB_MENU    = "title" of the kernel entry as displayed by: 'grubby --info=ALL'
+  GRUB_MENU    = ${LOCALVERSION}
```

Comment:
I don't see how this can work.

Reply:
A previous section has you set the title of the grub entry to `test`, which matches LOCALVERSION.

Reply:
My bad! Please keep the previous value command as a comment, it's a useful reference to have in the config file when something goes wrong. ktest variable names tend to be cryptic and `GRUB_MENU` is one of the worst.